### DISCUSSION TITLE
Can this find the seed of a world on a Realm/Server?

### DISCUSSION THREAD

**Discussion by KapDEK** [mircokroon/minecraft-world-downloader#276] [Q&A]
I was wondering if this is able to find the seed of a realm/server. I used the 1.18 pre release to download a realm, and it worked great, considering this is a pre release, but I was wondering if the finished releases get the seed, and will 1.18 get that when its finished? Thanks!

**Comment by mircokroon** [comment]
Unfortunately finding world seeds is a bit more difficult, there are tools available to do that if you look around online (though I imagine they don't work for 1.18 yet).
